### PR TITLE
Dashboard: Allow disabling dashboard grid lazy loading

### PR DESCRIPTION
**Pull request by dprokop**
A simple approach to allow all panels to load as soon as the dashboard is loaded. This allows to opt-out from when-in-viewport panel loading that is the current behavior. It's helpful for a scenario, when the data should be visible to the dashboard user as soon as possible (when scrolling).

I've also considered two alternatives:
1. Allow configuring the pixel threshold on when to load panels. I.e. load panels in view and panels in the next view viewport height.
2. Allow configuring the preload on the panel level.

Both of the approaches are a bit more complex to implement and I'm not certain if the complexity (both in the UI and code) is necessary. Open for discussion :)

for https://github.com/grafana/hyperion-planning/issues/57

### CODE REVIEW

File: public/app/features/dashboard-scene/settings/GeneralSettingsEditView.tsx [hunk 1]
```diff
@@ -271,6 +276,24 @@ export class GeneralSettingsEditView
             >
               <RadioButtonGroup onChange={model.onTooltipChange} options={GRAPH_TOOLTIP_OPTIONS} value={graphTooltip} />
             </Field>
+
+            <Field
+              label="Preload panels"
+              description="When enabled all panels will start loading as soon as the dashboard has been loaded"
```
Comment (ashharrison90) [inline public/app/features/dashboard-scene/settings/GeneralSettingsEditView.tsx:282]: we could mention there may be a performance impact to this 🤔

**Review by ashharrison90 — COMMENTED** [review]
if we decide to add this option, think this looks perfect! 🙌 (with the translation TODO ofc) 

there's a more general question of whether we should add this at all, or just tweak the `rootMargin` value [here](https://github.com/grafana/scenes/blob/main/packages/scenes/src/components/layout/LazyLoader.tsx#L77)

**Comment by dprokop** [comment]
@grafana/dashboards-squad / @torkelo  - just a friendly ping for attention

**Review by torkelo — APPROVED** [review]
Looks good 👍

File: public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts [hunk 2]
```diff
@@ -258,7 +259,7 @@ export function createDashboardSceneFromDashboardModel(oldModel: DashboardModel)
     uid: oldModel.uid,
     version: oldModel.version,
     body: new SceneGridLayout({
-      isLazy: true,
+      isLazy: dto.preload ? false : true,
```
Comment (ashharrison90) [inline public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts:262]: minor nit:
```suggestion
      isLazy: !dto.preload,
```

File: public/app/features/dashboard-scene/settings/GeneralSettingsEditView.tsx (hunk 1 above)
Comment (ashharrison90) [reply public/app/features/dashboard-scene/settings/GeneralSettingsEditView.tsx:282]: we should do the translations here,

basically as the example below then run `make i18n-extract`

File: public/app/features/dashboard-scene/settings/GeneralSettingsEditView.tsx [hunk 3]
```diff
@@ -271,6 +276,24 @@ export class GeneralSettingsEditView
...
+
+              // TODO
+              // label={t('dashboard-settings.general.panel-options-graph-tooltip-label', 'Graph tooltip')}
+              // description={t(
+              //   'dashboard-settings.general.panel-options-graph-tooltip-description',
+              //   'Controls tooltip and hover highlight behavior across different panels. Reload the dashboard for changes to take effect'
+              // )}
```
Comment (ashharrison90) [inline public/app/features/dashboard-scene/settings/GeneralSettingsEditView.tsx:289]: commented out code can be removed (assume this is a reference for adding the translations)

**Review by ashharrison90 — COMMENTED** [review]
translations should be added, otherwise looks great 👍

**Pull request merged**